examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3013,6 +3013,8 @@ class GLTFParser {
 
 		return this.getDependency( 'texture', mapDef.index ).then( function ( texture ) {
 
+			if ( ! texture ) return null;
```

Comment:
Similar to the WebXR related changes 😇 :
```js
if ( texture === undefined ) return null;
```

Reply:
BTW: Why isn't it possible for us collaborators to edit the changes in your PR? This normally works elsewhere 🤔 .

Reply:
> BTW: Why isn't it possible for us collaborators to edit the changes in your PR? This normally works elsewhere

I guess this checkbox in the PR is unchecked?

![image](https://user-images.githubusercontent.com/7637832/204080822-d63e600f-d60d-4e6b-b215-a8de1a1e42e4.png)

Reply:
Thanks, that's it! 

It would be great to always keep this UI always checked so we apply clean up edits to the PRs.

Reply:
I'll try that, sorry, wasn't aware!

I want to ask here why the check for undefined - if the texture object is null the code below would still fail with an exception. I thought the `if (texture)` check checks for both?

Reply:
True. I just wanted to clarify whether `texture` is `null` _or_ `undefined` when it isn't found under a given reference/index.

Reply:
Understood! So with the test case I had (added a few lines of code to original post), `getDependency( 'texture' )` returns null but I think that depends on which extension loader is actually used - these could either return null or undefined if I'm not mistaken depending on what went wrong.

https://github.com/mrdoob/three.js/blob/b44d661e4b1afe7dc299ee7ceba4c9fe69bb2c60/examples/jsm/loaders/GLTFLoader.js#L2576-L2581

Reply:
Never mind then^^. We can clean things up later, too. For now, the code is fine.

Reply:
`texture` should be null or texture instance here. If it is others, for example undefined, probably a ceratain plugin doesn't follow the plugin API.